assets/js/components/settings/SettingsActiveModule/FooterSecondaryAction.js

```diff
@@ -64,7 +64,7 @@ export default function FooterSecondaryAction( {
 		);
 	}
 
-	if ( ! isEditing && moduleHomepage ) {
+	if ( ! isEditing && module.connected && moduleHomepage ) {
```

Comment:
This seems like it could crash/error, but the fix is straightforward:

```suggestion
	if ( ! isEditing && module?.connected && moduleHomepage ) {
```